app/default/data.py

```diff
@@ -168,17 +175,21 @@ def get_applications_for_account(account_id, as_dict=False):
     )
 
 
-def get_fund_data(fund_id, language=None, as_dict=False):
+@lru_cache(maxsize=5)
+def get_fund_data(fund_id, language=None, as_dict=False, ttl_hash=None):
```

Comment:
For consistency, these should probably be separated onto their own lines, as [seen below](https://github.com/communitiesuk/funding-service-design-frontend/pull/366/files#diff-25ffeea551d125947892e07d498380fe030e5f7c005ba15e165622de5d567f0eR242)